### DISCUSSION TITLE
Add checkbox to opt-out of open client count

### DISCUSSION THREAD

**Discussion by jcstryker** [runelite/runelite#15051] [Ideas]
Currently the RuneLite client pings the RuneLite server every 4 minutes to provide the Open Client counter on the website and discord. Since this is not critical to the functionality of the client it would be nice for there to be an option to disable the server pings.

Blocking the RuneLite server using the hosts file will disrupt other client services like price checking, etc.

**Comment by Unmoon** [comment]
Probably a good addition if we start tracking price data and allow opt-out / opt-in for it too. #733

**Comment by Tylemagne** [comment]
I think opting out should be optional and enclosed in one single checkbox. Opting out of the counter and price checkers should all be wrapped into one simple click.

**Comment by kamielvf** [comment]
If you're worried about sending pings just run the api yourself? That way you have zero connections to the official RuneLite api and everything is local.

**Comment by deathbeam** [comment]
If you want price checking but not ping, then this request is completely invalid. We are offering multiple solution for no access to runelite api already. Hosts file, hosting your own API.

**Comment by jcstryker** [comment]
This isn't for no access to the runelite api, this is just for the open client pings, I don't understand that how this is invalid?

**Comment by jcstryker** [comment]
can you please reopen this @deathbeam, I don't expect this to be an immediate priority, but maybe as a beginner-friendly PR

**Comment by deathbeam** [comment]
I find this as not reasonable suggestion, but ok, w/e I guess.
